docs/dyndns/configure.rst

```diff
@@ -174,10 +174,15 @@ expected.
 
 Updating Multiple Domains
 `````````````````````````
-To update multiple domain or subdomains, it is best to designate one of them
-as the main domain, and create CNAME records for the others, so that they act
-as DNS aliases for the main domain.
-You can use do that either via the web interface or the API.
-
-If you try to update several subdomains directly (by issuing multiple update
-requests), your update requests may be refused (see :ref:`rate-limits`).
+To have multiple domain names point to the same dynamic IP address, it is best
+to designate one of them as the main domain for updates and create CNAME records
+for the others. This way, the other domains will automatically resolve to the same
+IP address as the main domain.
+
+Alternatively, for cases where a CNAME is not suitable (e.g. at the zone apex),
```

Comment:
If multiple names are to be updated (within the same domain), then only max. one of them is at the apex, so this can always be the main domain to which the others point via CNAME. So this is not a good example for where CNAME is not suitable. What would be a good use case?